app/src/main/java/com/iitdh/sonusourav/instigo/Account/LoginActivity.java

```diff
@@ -315,9 +316,16 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                         firebaseAuthWithGoogle(account);
                     }
                 } catch (ApiException e) {
-                    Toast.makeText(LoginActivity.this, e.getMessage(), Toast.LENGTH_SHORT).show();
+                    if (e.getStatusCode() == GoogleSignInStatusCodes.SIGN_IN_CANCELLED) {
+                        // cancelled
```

Comment:
Looks fine to me. Just remove the redundant comments and squash your commits in the last.